### ISSUE TITLE
Мобильная тема

### ISSUE THREAD

**Issue by xamgore** [mmcs-sfedu/PL_Lections_2014-2015#40]
В общем, раньше лекции очень хорошо отображались для мобильных браузеров. А теперь с появлением панельки – все плохо.

Предлагаю сделать в css @media для разных размеров экрана (copy-paste из bootstrap'a). С телефонов, правда, хочется читать.

**Comment by ghost** [comment]
Ну, в этих вопросах я не помощник. Во-первых я мало что смыслю в мобильных браузерах. Во-вторых мне не на чем тестировать.
Если самостоятельно сделаешь - будет здорово. Мне главное, чтобы с компьютера можно было нормально читать, по крайней мере на моих 10'' и 22'' все выглядит неплохо.

**Comment by ghost** [comment]
> copy-paste из bootstrap'a

Да, это конечно сделает текст читабельным, однако, боюсь, со временем захочется сделать выдвижной ТОС - на маленьких экранах он даже актуальнее.
